strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/Repository.java

```diff
@@ -58,13 +61,17 @@
     boolean acceptsSnapshots();
 
     boolean acceptsReleases();
-    
+
     RepositoryConfiguration getRepositoryConfiguration();
 
     String getStorageIdAndRepositoryId();
 
     boolean isType(String type);
 
     RemoteRepository getRemoteRepository();
-    
+
+    ProxyServerConfiguration getProxyServerConfiguration()
+        throws IllegalAccessException,
+        InvocationTargetException;
```

Comment:
Cosmetic: Could you please set your IDE to align exceptions? (I've seen this elsewhere across the pull request in methods as well). Thanks! :)

Reply:
@carlspring : Pls share the format, I am using the same template available on strongbox documentation page.